test(vagrantbox): automate vagrantbox tests CFY-3996

guy-mograbi-at-gigaspaces opened the pull request.

m1keil commented on quickstart/automated-test/jenkins/script.sh:
```diff
@@ -0,0 +1,65 @@
+clean_env
+
+VAGRANTFILE_URL="http://gigaspaces-repository-eu.s3.amazonaws.com/org/cloudify3/${VERSION}/${PRERELEASE}-RELEASE/Vagrantfile"
+wget -O Vagrantfile $VAGRANTFILE_URL
```
use `vagrant box add` instead

guy-mograbi-at-gigaspaces replied:
how can I use `vagrant box add` instead here?

guy-mograbi-at-gigaspaces replied:
also please note that this file follows instructions from quickstart which says download and run vagrant up

http://getcloudify.org/guide/3.2/quickstart.html

m1keil replied:
my bad, didn't notice this was vagrantfile you downloading

guy-mograbi-at-gigaspaces committed:
CFY-3966 add automated test for vagrantbox

adding script for automated test

adding script for automated test

adding script for automated test

adding script for automated test

guy-mograbi-at-gigaspaces committed:
make the tests see the vagrantbox by changing the baseurl.

guy-mograbi-at-gigaspaces committed:
use phantomjs instead of chrome

guy-mograbi-at-gigaspaces commented:
need to fix some stuff - and then I will reopen
